exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/handlers/MetastoreAnalyzeTableHandler.java

```diff
@@ -406,13 +406,13 @@ private DrillRel getTableAggRelNode(DrillRel convertedRelNode, boolean createNew
     SchemaPath lastModifiedTimeField =
         SchemaPath.getSimplePath(config.getContext().getOptions().getString(ExecConstants.IMPLICIT_LAST_MODIFIED_TIME_COLUMN_LABEL));
 
-    List<SchemaPath> excludedColumns = Arrays.asList(locationField, lastModifiedTimeField);
+    List<SchemaPath> nonSchemaColumns = Arrays.asList(locationField, lastModifiedTimeField);
```

Comment:
Again, here and elsewhere, please pick a more descriptive name.

Reply:
Thanks, renamed here and in other places.